mk/flags.mk

```diff
@@ -177,7 +177,7 @@ override ASFLAGS += $(asflags)
 override COMMON_CCFLAGS := $(COMMON_FLAGS)
 override COMMON_CCFLAGS += -fno-strict-aliasing -fno-common
 override COMMON_CCFLAGS += -Wall -Werror
-override COMMON_CCFLAGS += -Wundef -Wno-trigraphs -Wno-char-subscripts 
```

Comment:
That's strange, we introduced `@Cflags` workaround for this, and it's used in this file, https://github.com/embox/embox/pull/964/files#diff-9de35d930b1e5f85a4612a4c4130222fL9

Could you check or provide compiler invocation comand for this sources? It should have `... -Wundef ... -Wno-undef ...`, in this order